ArduCopter/Copter.h

```diff
@@ -635,6 +635,7 @@ class Copter : public AP_Vehicle {
     bool start_takeoff(float alt) override;
     bool set_target_location(const Location& target_loc) override;
     bool set_target_velocity_NED(const Vector3f& vel_ned) override;
+    bool set_target_angle_and_climbrate(float roll_deg, float pitch_deg, float yaw_deg, float climb_rate_cms, bool use_yaw_rate, float yaw_rate_rads) override;
```

Comment:
looks like it's got the argument named as "climb_rate_cms"

Reply:
I'm surprised this build at all

Reply:
Yeah, C++ only checks that the types match, not that the names of the arguments match.